jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -167,9 +167,9 @@ protected void generateAcceptableResponse(Request baseRequest, HttpServletReques
         {
             for (String mimeType : acceptable)
             {
-                generateAcceptableResponse(baseRequest, request, response, code, message, mimeType);
-                if (baseRequest.isHandled())
+                if (generateAcceptableResponse(baseRequest, request, response, code, message, mimeType)) {
```

Comment:
please apply coding style so that { is on a new line

Reply:
Fixed